tea-config-generator/serverless.yml

```diff
@@ -37,11 +36,11 @@ functions:
   status:
     handler: handler.health
     description: Return a nominal response for confirming the status of this app
-    timeout: 1
+    timeout: 5
     events:
       - alb:
           listenerArn: ${cf:${self:provider.stage}.servicesLbListenerArn}
-          priority: 51
+          priority: 998
           conditions:
             path: ${self:custom.endPoint}/status
     vpc: ~
```

Comment:
What does `~` mean here? We should be able to delete this line.

Reply:
it means use the default, I was using it to document that I mean to not apply the setting at this level because it is not needed. If this is not valuable, then I can remove it. I would be required if I moved the vpc setting to the provider level.

Reply:
I am not sure what the standard is. I will leave it up to you to keep it or delete it.